Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Valeri Karpov <valkar207@gmail.com>",
            "Saghm Rossi <saghmrossi@gmail.com>",
            "Kevin Yeh <kevinyeah@utexas.edu>"]
 
-description = "An experimental MongoDB driver written by MongoDB interns."
+description = "A native-Rust MongoDB driver. Originally written by MongoDB interns, the driver is an unofficial but still actively maintained project."
```

Comment:
In terms of the copyright, I'm pretty sure it's owned by MongoDB since we wrote it while employed by the company, and in terms of the Github repo, it certainly is! The Go driver is in a bit different situation; the person who wrote it used to be contracted by MongoDB to maintaining it, but that's no longer the case, hence the disparity between "developed under" and "maintained by".

Working on this driver is explicitly not part of my job or related to my employment with MongoDB,  but I'm not exactly maintaining it  "as a community member" in the sense that "community" is generally by MongoDB to describe drivers. Honestly, I think that this driver is in a fairly unique position that the usual terminology that MongoDB uses to describe drivers isn't really that well-equipped to handle, which is part of why I originally suggested just saying exactly what the origins and current state of the driver is, i.e. created by MongoDB interns, not officially supported but actively maintained. If you still aren't fond of keeping the "created by MongoDB interns" part, I'd be all right with something similar to your suggestion in the first comment. I'd want to tweaking the language, as at least to me, it sounds like it's saying the driver doesn't officially support MongoDB 3.0+, which isn't the case; the driver definitely supports MongoDB the database, it's just that MongoDB the company doesn't provide official support for the driver.